data/test/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(intergrationTest)
```

Comment:
Not a comment on this file specifically, but you have a typo in the path data/test/intergrationTest - extra 'r' in integration

Reply:
WOOPS